interface-definitions/service_dns_forwarding.xml.in

```diff
@@ -793,6 +793,155 @@
                   </leafNode>
                 </children>
               </node>
+              <tagNode name="zone-to-cache">
```

Comment:
I really don't like the name PowerDNS chose for that function and I don't think we should mimic it. Just `zone-cache` would suffice.

Reply:
done